scripts/patched-jest-resolver.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+Object.defineProperty(exports, '__esModule', { value: true });
+const path_1 = require('path');
+const ts = require('typescript');
+const defaultResolver_1 = require('jest-resolve/build/defaultResolver');
+function getCompilerSetup(rootDir) {
+  const tsConfigPath =
+    ts.findConfigFile(rootDir, ts.sys.fileExists, 'tsconfig.spec.json') ||
+    ts.findConfigFile(rootDir, ts.sys.fileExists, 'tsconfig.test.json') ||
+    ts.findConfigFile(rootDir, ts.sys.fileExists, 'tsconfig.jest.json');
+  if (!tsConfigPath) {
+    console.error(
+      `Cannot locate a tsconfig.spec.json. Please create one at ${rootDir}/tsconfig.spec.json`
+    );
+  }
+  const readResult = ts.readConfigFile(tsConfigPath, ts.sys.readFile);
+  const config = ts.parseJsonConfigFileContent(
+    readResult.config,
+    ts.sys,
+    path_1.dirname(tsConfigPath)
+  );
+  const compilerOptions = config.options;
+  const host = ts.createCompilerHost(compilerOptions, true);
+  return { compilerOptions, host };
+}
+let compilerSetup;
+module.exports = function (path, options) {
+  const ext = path_1.extname(path);
+  if (
+    ext === '.css' ||
+    ext === '.scss' ||
+    ext === '.sass' ||
+    ext === '.less' ||
+    ext === '.styl'
+  ) {
+    return require.resolve('identity-obj-proxy');
+  }
+  // Try to use the defaultResolver
+  try {
+    if (path.indexOf('@nrwl/workspace') > -1) {
```

Comment:
This file is copied from `@nrwl/jest/plugins/resolver` - with the exception that we `throw` if the `@nrwl/workspace` is in the path: if any of the tests reference this package, we want to use the local version instead (`./packages/workspace`), that's mapped in the root `tsconfig.json` - rather than the default one in `node_modules`.

This is a temporary fix, Victor said he'll look into a more permanent solution.